test/integration/admin_can_see_admin_dashboard_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'test_helper'
+
+class AdminCanSeeAdminDashboardTest < ActionDispatch::IntegrationTest
+  test "admin can view admin dashboard" do
+    admin = make_admin
+
+    ApplicationController.any_instance.stubs(:current_user).returns(admin)
+    visit '/admin/dashboard'
+
+    assert page.has_content?("Admin Dashboard")
+  end
+
+  test "registered user cannot view admin dashboard" do
+
+    logged_in_user
+    visit '/admin/dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.